src/UnifiedFFI/FFICompilerPlugin.class.st

```diff
@@ -41,40 +41,75 @@ Class {
 }
 
 { #category : #adding }
-FFICompilerPlugin class >> addAFfiCalloutSelectorEvent: aPragmaAdded [
-	self addAFfiCalloutSelectorFromPragma: aPragmaAdded pragma
+FFICompilerPlugin class >> addFFICalloutSelector: aString [ 
+	FFICalloutSelectors add: aString
 ]
 
 { #category : #adding }
-FFICompilerPlugin class >> addAFfiCalloutSelectorFromPragma: aPragma [
-	self addFfiCalloutSelector: aPragma methodSelector.
-	self recompileSendersOf: aPragma method
+FFICompilerPlugin class >> addFFICalloutSelectorEvent: aPragmaAdded [
+	self addFFICalloutSelectorFromPragma: aPragmaAdded pragma
 ]
 
 { #category : #adding }
-FFICompilerPlugin class >> addFfiCalloutSelector: aString [ 
-	FFICalloutSelectors add: aString
+FFICompilerPlugin class >> addFFICalloutSelectorFromPragma: aPragma [
+	self addFFICalloutSelector: aPragma methodSelector.
+	self recompileSendersOf: aPragma method
+]
+
+{ #category : #'private - defaults' }
+FFICompilerPlugin class >> defaultFFICalloutSelectors [
+	"FFI uses UFFI and NB for the time being"
+	
+	^ self defaultUFFICalloutSelectors 
+		join: self defaultNativeBoostCalloutSelectors 
```

Comment:
I wonder that #join really works for two arrays. But it make strange order. 
I think it is better to use #, message because the purpose of join: is different even if it works here.